modules/busco/main.nf

```diff
@@ -0,0 +1,48 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process BUSCO {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::busco=5.2.2" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/busco:5.2.2--pyhdfd78af_0"
+    } else {
+        container "quay.io/biocontainers/busco:5.2.2--pyhdfd78af_0"
+    }
+
+    input:
+    tuple val(meta), path(fasta)
+    path(augustus_config)
+
+    output:
+    tuple val(meta), path("${meta.id}/run_*/full_table.tsv"), emit: tsv
+    tuple val(meta), path("${meta.id}/run_*/short_summary.txt"), emit: txt
```

Comment:
please use `prefix` instead:
```suggestion
    tuple val(meta), path("${prefix}/run_*/full_table.tsv"), emit: tsv
    tuple val(meta), path("${prefix}/run_*/short_summary.txt"), emit: txt
```
Also, output should be detailed, each relevant file in one channel, also see [here](https://nf-co.re/developers/adding_modules#inputoutput-options) (it probably doesn't need to be compressed, as long as the ouput is quite small)